bitnami/nginx/templates/deployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
       {{- end }}
     spec:
       {{- include "nginx.imagePullSecrets" . | nindent 6 }}
+      automountServiceAccountToken: false
```

Comment:
I would use the same approach as in #6967 and add a new parameter in the `ServiceAccount` section of the `values.yaml`, then bump the minor version of the chart. What do you think?

Reply:
Sound good, I will amend this